core/embed/rust/src/ui/shape/display/direct_canvas.rs

```diff
@@ -0,0 +1,24 @@
+use crate::ui::{
+    display::Color,
+    shape::{Canvas, DirectRenderer, DrawingCache},
+};
+
+use super::bumps::run_with_bumps;
+
+/// Creates the `Renderer` object for drawing on a the specified and invokes a
```

Comment:
on a the?

Reply:
fixed, see 2979759b390cd2f3188e103602e5a6449a9e66ea